src/global/config.ts

```diff
@@ -1,3 +1,5 @@
+import { documentLanguage } from './eventing';
```

Comment:
Maybe I do like this approach better because it keeps the format and dateParser functions together, and that makes sense since they're likely to be used together. My only objection might be that this would make it harder to define different cutoffYearOffsets for different Datepicker instances, but it still looks like a really niche use case, so the tradeoff is probably still worth it.

Reply:
I think that is an acceptable tradeoff, as I think it would actually be confusing to have different cutoffYearOffsets in one app.
So I agree with you that it would be really niche and I think we can ignore it for now.
Feel free to go ahead with the refactoring, if you agree with it. 👍